### DISCUSSION TITLE
Errors with npm run start

### DISCUSSION THREAD

**Discussion by rishiskhare** [AXeL-dev/distract-me-not#98] [Q&A] [answered]
I just forked this repo, and I installed all dependencies. However, I keep getting this error when running npm run start:
<img width="746" alt="image" src="https://github.com/AXeL-dev/distract-me-not/assets/30673002/790a35ca-b99e-427e-86ad-775bb17475e9">

Does anyone know how to resolve this? I've tried reinstalling Node and reinstalling dependencies, all with no luck.

**Comment by AXeL-dev** [comment, marked as answer]
You should downgrade your node version, i suggest you to use nvm to do so, please follow these steps:

- Download [nvm](https://github.com/nvm-sh/nvm) or [nvm-windows](https://github.com/coreybutler/nvm-windows) (depending on your OS) & install it.
- Install node version `16` & use it:
```
nvm install 16
nvm use 16
```
- Open a new terminal in the **DistractMeNot** project & run the `npm start` command.
